src/modules/paymentProcessor/PP_Streaming_v1.sol

```diff
@@ -101,13 +101,14 @@ contract PP_Streaming_v1 is Module_v1, IPP_Streaming_v1 {
     ///         client => paymentReceiver => arrayOfStreamIdsWithPendingPayment(uint[]).
     mapping(address => mapping(address => uint[])) private activeStreams;
 
-    /// @dev    Default start, cliff and end times for new payment orders.
-    uint private defaultStart;
-    uint private defaultCliff;
-    uint private defaultEnd;
+    /// @dev    The flags that this PaymentProcessor uses
+    bytes32 internal constant processor_flags =
+        0x000000000000000000000000000000000000000000000000000000000000000e; // 1110
```

Comment:
I'd not add stuff at the end as the line is too ling already anyways and it would not be in the docs otherwise if we generate these from the NatSpec.

```suggestion
    /// @dev    The flags that this PaymentProcessor uses.
    ///         Contains the value 1110.
    bytes32 internal constant processor_flags =
        0x000000000000000000000000000000000000000000000000000000000000000e;
```